packages/grafana-data/src/types/datasource.ts

```diff
@@ -79,6 +80,9 @@ export class DataSourcePlugin<
     return this;
   }
 
+  /*
+   * @deprecated -- prefer using {@link CustomVariableSupport or DataSourceVariableSupport}
```

Comment:
```suggestion
   * @deprecated -- prefer using {@link CustomVariableSupport} or {@link DataSourceVariableSupport}
```

not 100% but I think we need to change the syntax like this.

Reply:
Ok, will look into it